.github/CODEOWNERS

```diff
@@ -0,0 +1,98 @@
+#    based.
+#
+# ------------------------------------------------------------------------------
+# Information for github
+# ------------------------------------------------------------------------------
+# These people are the default "owners" for everything in the repo unless a
+# later match is made, they will automatically be requested to review PRs.
+* @sklam @stuartarchibald @esc
+
+# Owners of specific parts of the code, will be requested to review if a PR
+# touches code in the matched pattern
+/numba/cuda @gmarkall
+/numba/parfors @DrTodd13
+/numba/stencils @DrTodd13
+
+# ------------------------------------------------------------------------------
+# Information for contributors
+# ------------------------------------------------------------------------------
+# This section provides a rough list of who to contact first for help with
+# various parts/concepts in the code base, first contact does not imply
+# ownership!
+#
+# Parts of the code base:
+#
+# * Parfors/Parallel Accelerator (@DrTodd13)
+#       - Array Analysis (@DrTodd13)
+#       - Parfors transforms (@DrTodd13)
+# * Stencils (@DrTodd13)
+# * Experimental:
+#       - Jitclasses (@sklam)
+#       - StructRef (@sklam)
+# * Typed containers:
+#       - Typed.List (@esc)
+#       - Typed.Dict (@sklam)
+# * Documentation (Needs first contact/owner)
+# * NumPy (Needs first contact/owner)
+#       - ufuncs (Needs first contact/owner)
+#       - linalg (@stuartarchibald)
+#       - Implementation of specific functions (Needs first contact/owner)
+#       - Parallel backends/threading layers (@stuartarchibald)
+# * CPython implementation (Needs first contact/owner)
+# * Extension API (Needs first contact/owner)
+# * AOT (Needs first contact/owner)
+# * Compiler:
+#       - Type inference (@sklam)
+#       - Bytecode analysis/CFA/DFA (@sklam)
+#       - Compiler Pipeline infrastructure (@stuartarchibald)
+#       - Compiler passes:
+#               - Rewrites (Needs first contact/owner)
+#               - Branch pruning (@stuartarchibald)
+#               - Literal unroll (@stuartarchibald)
+#               - Rewrite Semantic Constants (@stuartarchibald)
+#               - MakeFunction To Jit function (@stuartarchibald)
+#               - Overload and function inlining (@stuartarchibald)
+#               - With Lifting (@sklam)
+#               - Exception handling (@sklam)
+#               - Literally (@sklam)
+#               - SSA (@sklam)
+#               - lowering.py, codegen.py (@sklam)
+#       - Datamodels/call conventions (@sklam)
+#       - Inlining in general (@stuartarchibald)
+#
+# Additional Concepts:
+#
+# * Reference counting and NRT (@sklam)
+# * Testing (Needs first contact/owner)
+# * CI:
+#       - public CI (azure) (Needs first contact/owner)
+#       - Numba build farm (@esc)
+# * Integration testing (https://github.com/numba/numba-integration-testing)
+#   (@esc)
+# * ASV profiling (@esc)
+# * Type Annotations (Needs first contact/owner)
```

Comment:
@EPronovost @luk-f-a would you be interested in this? If so will add. Thanks.

Reply:
sure, I'm happy to be added.

Reply:
Same

Reply:
Thanks both for confirming, added in 8d94a12.